gamedata/configs/text/eng/srp_strings.xml

```diff
@@ -10,13 +10,13 @@
 		<text>Nowhere, I changed my mind.</text>
 	</string>
 	<string id="st_tm_guide_213">
-		<text>Whoops, I can�t afford this.</text>
+		<text>Seems like I can’t afford this.</text>
```

Comment:
Not a fan of this change; it fails to convey the surprise of the original.

Context: From what I remember, this dialog is a work-around for a bug in the vanilla game that enables the player to select a fast-travel point, break out of the dialog, go spend some money, re-enter the dialog, and re-select the travel point even if it can no longer be afforded.

The dialog is meant to purport that the _player_ has mistakenly selected the unaffordable travel point, even though it's actually the game's programming that has mistakenly made it possible to re-select.

I don't think this dialog needs changing, but if you must, a better one would be:
>My mistake - I can’t afford this.

Reply:
I don't really know is these things existed in English localization, but in Russian 'Whoops' sounded just wrong for a man like Scar. Most of his dialogues leave an impression of an man in his fourties with an always straight face. Probably just my portrait of him and I might be wrong, but there's never something like 'Whoops' in the original game. I recall we discussed this in Steam chats, and you did not oppose this change back then.

> My mistake - I can’t afford this.
Yeah, I like that compromise as well. It’s something that’s kinda in-between.